src/tests/tests/firstparties.js

```diff
@@ -0,0 +1,71 @@
+(function() {
+  let tco = 'http://t.co/beach-detour/';
+  let destination = 'https://the.beach/';
+
+  document.querySelectorAllBefore = document.querySelectorAll;
```

Comment:
I added a stub for `setInterval`. I don't think the stubs should be added to `before`/`beforeEach` since they are specific to one test (not every test, or the module).